docs/AddingCustomOp.md

```diff
@@ -8,6 +8,7 @@ Note: These APIs are experimental and will change in the next release. They're r
 * Create an OrtCustomOp structure for each op and add them to the OrtCustomOpDomain with OrtCustomOpDomain_Add
```

Comment:
I guess these APIs are not experimental any more since they're now part of our official header file. The header file has END EXPERIMENTAL, but no BEGIN EXPERIMENTAL :)

Reply:
Should I remove END EXPERIMENTAL ?

Reply:
Nah, let's do it in a separate PR as this is the not the objective of this PR.